code/Model/SiteTreeLinkTracking_Parser.php

```diff
@@ -91,7 +92,7 @@ public function process(SS_HTMLValue $htmlValue)
 					'Target' => $matches['id'],
 					'Anchor' => null,
 					'DOMReference' => $link,
-					'Broken' => !DataObject::get_by_id('File', $matches['id'])
+					'Broken' => !DataObject::get_by_id('SilverStripe\\Assets\\File', $matches['id'])
```

Comment:
We could take this as an opportunity to rewrite this all to the less verbose `File::get()->byID()` ... but then again, that's annoyingly not cached in the same way as `get_by_id()`